docs/guide/yaml/example_yaml/test-app-with-enrichers-slightly-simpler.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# example showing how enrichers can be set 
+# 
+name: test-app-with-enrichers
+description: Testing many enrichers
+services:
+- type: brooklyn.entity.group.DynamicCluster
+  id: cluster
+  initialSize: 3
+  location: localhost
+  memberSpec:
+    $brooklyn:entitySpec:
+      type: brooklyn.test.entity.TestEntity
+      brooklyn.enrichers:
+      - type: brooklyn.enricher.basic.Transformer
+        # transform "ip" (which we expect a feed, not shown here, to set) to a URL;
+        # you can curl an address string to the sensors/ip endpoint an entity to trigger these enrichers 
+        brooklyn.config:
+          enricher.sourceSensor: $brooklyn:sensor("ip")
+          enricher.targetSensor: $brooklyn:sensor("url")
+          enricher.targetValue: $brooklyn:formatString("http://%s/", $brooklyn:attributeWhenReady("ip"))
```

Comment:
Great examples, and great that we can now support all of these. I like the yaml here.

Rambling thoughts follow, not pertinent to this PR!

I wonder about things like `$brooklyn:sensor("brooklyn.entity.basic.Attributes", "main.uri")`. As long as people hardly ever write it, then I'm ok. Otherwise, need to think of the trade-off for letting them just create a new untyped sensor with the same name, versus it finding the statically defined sensor. The untyped obviously wouldn't have the description and type metadata, but would be simpler in the yaml.